Lib/idlelib/iomenu.py

```diff
@@ -237,6 +239,49 @@ def save_a_copy(self, event):
         self.updaterecentfileslist(filename)
         return "break"
 
+    def reload(self, event):
+        """Reload the file from disk, discarding any unsaved changes.
+
+        If the file has unsaved changes, ask the user to confirm.
+        """
+        if not self.filename:
+            messagebox.showinfo(
+                "No File",
```

Comment:
```suggestion
                “File Not Found",
```
I think File Not Found would be better because it is a more descriptive version of “No File” in this case